src/MAUI/Maui.Samples/Samples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.xaml

```diff
@@ -1,90 +1,95 @@
-                   TextColor="White" />
-        </Grid>
-        <StackLayout x:Name="BusyIndicator"
-                     Grid.Row="0"
-                     Grid.RowSpan="2"
-                     HeightRequest="40"
-                     IsVisible="False"
-                     VerticalOptions="Center">
-            <ActivityIndicator HeightRequest="40"
-                               HorizontalOptions="Center"
-                               IsRunning="True"
-                               VerticalOptions="Center">
-                <!--<ActivityIndicator.Color>
-                    <OnPlatform x:TypeArguments="Color" iOS="#2499CE" />
-                </ActivityIndicator.Color>-->
-            </ActivityIndicator>
+﻿<ContentPage
+    x:Class="ArcGISRuntime.Samples.TraceUtilityNetwork.TraceUtilityNetwork"
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Maui;assembly=Esri.ArcGISRuntime.Maui">
+    <Grid Style="{DynamicResource EsriSampleContainer}">
+        <esriUI:MapView
+            x:Name="MyMapView"
+            GeoViewTapped="OnGeoViewTapped"
+            Style="{DynamicResource EsriSampleGeoView}" />
+        <StackLayout
+            x:Name="BusyIndicator"
+            Grid.Row="0"
+            Grid.RowSpan="2"
+            HeightRequest="40"
+            IsVisible="False"
+            VerticalOptions="Center">
+            <ActivityIndicator
+                HeightRequest="40"
+                HorizontalOptions="Center"
+                IsRunning="True"
+                VerticalOptions="Center" />
         </StackLayout>
-        <Grid x:Name="PickerUI"
-              Grid.Row="0"
-              IsVisible="False">
+        <Grid
+            x:Name="PickerUI"
+            Grid.Row="0"
+            IsVisible="False">
             <Grid.RowDefinitions>
                 <RowDefinition Height="auto" />
                 <RowDefinition Height="*" />
             </Grid.RowDefinitions>
-            <Label Grid.Row="0"
-                   Margin="5"
-                   HorizontalTextAlignment="Center"
-                   Text="Choose a terminal for this junction." />
-            <ListView x:Name="TerminalPicker"
-                      Grid.Row="1"
-                      ItemSelected="Terminal_Selected" />
+            <Label
+                Grid.Row="0"
+                Margin="5"
+                HorizontalTextAlignment="Center"
+                Text="Choose a terminal for this junction." />
+            <ListView
+                x:Name="TerminalPicker"
+                Grid.Row="1"
+                ItemSelected="Terminal_Selected" />
         </Grid>
-        <StackLayout x:Name="MainUI"
-                     Grid.Row="2"
-                     Orientation="Vertical">
-            <Button x:Name="TracePickerButton"
+        <Border x:Name="ControlPanelBorder" Style="{DynamicResource EsriSampleControlPanel}">
```

Comment:
Width changes when button text/label text changes in horizontal orientation. Fixed/max width would prevent this.